openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -510,13 +510,10 @@ export const addUser = (username, email) => {
     .should('be.visible')
     .type('Adding user');
   interceptURL('GET', ' /api/v1/users/generateRandomPwd', 'generatePassword');
-  cy.get('[data-testid="password-generator"]')
-    .should('be.visible')
-    .click();
+  cy.get('[data-testid="password-generator"]').should('be.visible').click();
   verifyResponseStatusCode('@generatePassword', 200);
-  interceptURL('POST', ' /api/v1/users', 'add-user');
+  cy.wait(1000);
```

Comment:
We have added the intercepter above so do we need this static wait?

Reply:
We have to add this as it's failing in CI. It's a workaround for now will remove this later